params/dhcpd-server.conf

```diff
@@ -4,6 +4,6 @@
 # Define range of addresses that the server can assign to other devices
 subnet 192.168.2.0 netmask 255.255.255.0 {
 	max-lease-time 60;
-	range 192.168.2.1;
+	range 192.168.2.10 192.168.2.20;
```

Comment:
WHY?

Reply:
When I got the dvl it had no support for static IP Address. Keeping it at 192.168.2.1 made it receive the same address I had on the topside, and made me spend a few hours trying to understand what was going on.

Reply:
They do support static IP address now, though, so we can take this out